src/OrchardCore.Modules/OrchardCore.Forms/Assets/js/formsWidgetEditor.js

```diff
@@ -0,0 +1,55 @@
+formsWidgetEditor = function () {
```

Comment:
it's called from event why not just keep it inside event instead of making global function?

Reply:
Because it’s being used in multiple places like TextAreaPart, InputPart and SelectPart

Reply:
I’ll revisit this once I create part for the label options and another for the validation options.